index.html

```diff
@@ -106,7 +106,7 @@ <h1 class="brackets-logo">
         <div class="large-12 columns">
             <h1><span data-i18n="index.page.hero.description">A modern, open source text editor that understands web design.</span></h1>
             <div id="download">
-                <a id="hero-cta-button" href="https://github.com/adobe/brackets/releases/latest" class="large rounded radius button"><div data-i18n="[html]index.page.hero.download">Download Brackets <d id="download-brackets-version">1.1</d></div>
+                <a id="hero-cta-button" href="https://github.com/adobe/brackets/releases/latest" class="large rounded radius button"><div data-i18n="[html]index.page.hero.download">Download Brackets <d id="download-brackets-version">1.2</d></div>
```

Comment:
@ryanstewart If we did a banner, any thoughts on how to message this?  Maybe something like: "Sit tight! The Linux build of 1.2 is slightly delayed, but check back next week and we will likely have a new build available."

It's a drag that we can't do OS-specific update notifications.  Linux users will see a notification days before it's actionable, and then will have to manually check back on the website repeatedly to find when the actual update is available later.  I wonder if it might be simpler to just delay the release a few more days to avoid having things be so out of sync...